{{ cookiecutter.project_name_dashed }}/LICENSE

```diff
@@ -186,7 +186,7 @@
       same "printed page" as the copyright notice for easier
       identification within third-party archives.
 
-   Copyright {{ cookiecutter.year}} {{ cookiecutter.author_name}}
+   Copyright {% now "utc", "%Y" %} {{ cookiecutter.author_name}}
```

Comment:
What's the behavior if this is updated by Cruft? Would it update the year? If so, we probably don't want that, right?

Probably, we would want either:
1. The year the project was started
2. A range consisting of the year the project was started _and_ the current year (e.g., `2021-2024`)

Reply:
Perhaps a better solution would be to use a function to set the default according to the current year, but still allow users to change. It would make sure that the year is stable even with updates, and it would also allow users to start projects in the past.

Reply:
done